src/lib/containers/StackChanger.tsx

```diff
@@ -0,0 +1,81 @@
+import { isEqual } from 'lodash-es'
+import * as React from 'react'
+import { useEffect, useState } from 'react'
+import UniversalCookies from 'universal-cookie'
```

Comment:
Reason for choosing a cookie persistence store vs local storage?

Reply:
No reason, just didn't think about it! localStorage would be better